### ISSUE TITLE
Output rollup results to another object

### ISSUE THREAD

**Issue by afawcett** [SFDO-Community/declarative-lookup-rollup-summaries#111]
From a blog comment...

This looks great. But we would like to use it a little differently.... 
We are trying to setup a bunch of rollup rules on a given dataset that would populate an object that contains the results of all those rollups (with a label that matches the rollup name) 
In other words have a parent child relationship doesnt fit our needs, we want to just run the rollups against a dataset and as a result "build" a new dataset containing all the results. 

Is there a way we can get the current release to work for us in that way. I can send you some examples if that helps. 
Thanks

**Comment by afawcett** [comment]
What a great idea!!! Once Salesforce makes available [Flow Triggers](https://help.salesforce.com/HTViewHelpDoc?id=creating_workflow_flow_actions.htm&language=en_US) this ought to be possible to capture updates the tool makes to the parent record and insert / update another object holding the summary.

**Comment by gotters** [comment]
Yes that makes a lot of sense. Although our scenario is one where we dont have a parent record. We simply have a full data set holding all the detailed data. We then want to expose a tool to the user for them to build their own Summary KPI Rules. Running those rules would populate the kpi summary table which would be used by other parts of the system to track performance against those kpis.
So it would still be good to have a way to push directly into this summary table without the need for parent records.

**Comment by afawcett** [comment]
Ah yes, got it, thanks! :+1:

**Comment by wes1278** [comment]
Simply stated, this would be completely awesome. +1 for this enhancement.

**Comment by aheber** [comment]
This sounds like a variation on Analytic Snapshots.

**Comment by JodieM** [comment]
Nice use case.
